handler/src/test/java/io/netty/handler/ssl/SslHandlerTest.java

```diff
@@ -63,4 +67,52 @@ public void testNonByteBufNotPassThrough() throws Exception {
 
         ch.writeOutbound(new Object());
     }
+
+    private static class TlsReadTest extends ChannelOutboundHandlerAdapter {
+        public volatile boolean readIssued;
+
+        @Override
+        public void read(ChannelHandlerContext ctx) throws Exception {
+            readIssued = true;
+            super.read(ctx);
+        }
+
+        public void test(final boolean dropChannelActive) throws Exception {
+          SSLEngine engine = SSLContext.getDefault().createSSLEngine();
```

Comment:
I was thinking about using `SslContextBuilder` here, but decided to stay aligned with the rest of the tests in this file and use `SSLContext.getDefault()`. I hope that's ok, because it doesn't really matter what engine is used in the context of this PR and two new tests.